doc/tests/block.rst

```diff
@@ -0,0 +1,13 @@
+``block``
+=========
+
+.. versionadded:: 1.23
+    The ``block`` test was added in Twig 1.23.
+
+``block`` checks if a block is defined in the current context of the template.
+
+.. code-block:: jinja
+
+    {% if 'title' is block %}
```

Comment:
Additionally the ``has_block`` should check the existence of a block in the current template as same as [Twig_Template::hasBlock](https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Template.php#L253) and not the existence of a block in the context.

The trouble with ``is block`` is similar to ``is constant``. More correct would be:
```twig
{% if post.status is same as(constant('Post::PUBLISHED')) %}
```